{"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"kind":"identifier","text":"SMIChallengeReasonRefresh"}],"languages":["occ"]}]}],"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason"]]},"sections":[],"metadata":{"roleHeading":"Enumeration Case","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"SMIChallengeReasonRefresh"}],"fragments":[{"kind":"identifier","text":"SMIChallengeReasonRefresh"}],"externalID":"c:@E@SMIChallengeReason@SMIChallengeReasonRefresh","modules":[{"name":"SMIClientCore"}],"symbolKind":"case","title":"SMIChallengeReasonRefresh"},"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonrefresh"]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"abstract":[{"type":"text","text":"The verification token from the server has expired and user verification is required again."}],"kind":"symbol","identifier":{"url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonRefresh","interfaceLanguage":"occ"},"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason/SMIChallengeReasonRefresh":{"kind":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonRefresh","type":"topic","url":"\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonrefresh","abstract":[{"type":"text","text":"The verification token from the server has expired and user verification is required again."}],"fragments":[{"kind":"identifier","text":"SMIChallengeReasonRefresh"}],"title":"SMIChallengeReasonRefresh","role":"symbol"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"kind":"symbol","type":"topic","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","url":"\/documentation\/smiclientcore","title":"SMIClientCore","role":"collection","abstract":[]},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason":{"fragments":[{"kind":"identifier","text":"SMIChallengeReason"}],"title":"SMIChallengeReason","kind":"symbol","type":"topic","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason","abstract":[{"text":"The reason for a credential challenge. Possible values:","type":"text"}],"role":"symbol","url":"\/documentation\/smiclientcore\/smichallengereason"}}}